File: typescript/src/contracts/interfaces/iprocess_engine_client.ts
import {ClientAspects} from './client_aspects/index';

export interface IProcessEngineClient
  extends ClientAspects.IEmptyActivityClient,
  ClientAspects.IEventClient,
  ClientAspects.IManualTaskClient,
  ClientAspects.INotificationClient,
  ClientAspects.IProcessModelClient,
  ClientAspects.IUserTaskClient {}

File: dotnet/src/contracts/interfaces/IProcessEngineClient.cs
namespace ProcessEngineClient.Contracts
{
    using ProcessEngineClient.Contracts.ClientAspects;

    /// <summary>
    /// API for starting and managing BMPN processes.
    /// </summary>
    public interface IProcessEngineClient :
        IEmptyActivityClient,
        IEventClient,
        IManualTaskClient,
        IProcessModelClient,
        IUserTaskClient
    { }
}
